packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -164,6 +164,7 @@ class AnimationController extends Animation<double>
   @override
   double get value => _value;
   double _value;
+
```

Comment:
why the newline? it's part of the value "block"...

(i'm fine with it either way, but that's why it didn't have a newline before)